client/src/redux/fetch-user-saga.js

```diff
@@ -19,11 +19,7 @@ function* fetchSessionUser() {
       data: { user = {}, result = '' }
     } = yield call(getSessionUser);
     const appUser = user[result] || {};
-
-    const [userId] = Object.keys(user);
-
-    const sound = user[userId].sound;
-
+    const sound = store.get('fcc-sound') ?? false;
     store.set('fcc-sound', sound);
```

Comment:
```suggestion
```

I don't think we need to set it here anymore, since we're not pulling it from the user record.

Reply:
done